win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -998,6 +998,139 @@ static DWORD WINAPI chaos_thread_with_timers_func(LPVOID lpThreadParameter)
     return 0;
 }
 
+//static DWORD WINAPI chaos_thread_with_timers_no_lock_func(LPVOID lpThreadParameter)
```

Comment:
The easier way to have this test disabled is to have an XTEST_FUNCTION defined which does not expand to the ctest code.

Look for it in the code base, you will find some examples.
The advantage is that in that case the code gets compiled and we get checks on the fact that this actually compiles Ok, even though the test does not get to run.